### PR TITLE
README.md : update links, fix typos

### PR DESCRIPTION
**Pull request by SpiritLevel**
README.md : update links and remove dokuwiki reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +14,9 @@ For adding functionality or editing the style of the documentation see the
 To set up a test instance:
 
  * Clone this repository to a LAMP server.
- * Install [Composer](http://doc.silverstripe.org/framework/en/installation/composer)
+ * Install [Composer](http://docs.silverstripe.org/en/getting_started/composer/)
```
Comment (camfindlay) [inline README.md]: We tend to favour no / at the end (just a preference thing).

File: README.md [hunk 2]
```diff
@@ -43,13 +45,14 @@ To contribute an improvement to the doc.silverstripe.org functionality or
...
-a "docs" folder (for example, the framework documentation is stored at
-[github.com/silverstripe/sapphire](https://github.com/silverstripe/sapphire/tree/master/docs)).
+The content for doc.silverstripe.org is stored in the modules
+repository inside a "docs" folder (for example, the framework
+documentation is stored at
+[https://github.com/silverstripe/silverstripe-framework/tree/master/docs](https://github.com/silverstripe/silverstripe-framework/tree/master/docs).
 
 If you wish to edit the documentation content, submit a pull request on that
 Github project. Updates to the content are synced regularly with
-docs.silverstripe.org via a cron job.
```
Comment (camfindlay) [inline README.md]: This was already correct. The site is at docs.ss.org

File: README.md [hunk 3]
```diff
@@ -59,10 +62,3 @@ for each module from git and rebuilds the search indexes.
 
 	05 * * * * sites make -f /sites/ss2doc-v2/www/Makefile -C /sites/ss2doc-v2/www update
```
Comment (camfindlay) [inline README.md]: @ss23 is this still the correct cron on our docs server? If not can you drop in the correct one for future reference :)

**Comment by SpiritLevel** [comment]
@camfindlay It would be clearer to be able to use silverstripe-documentation for the repo and docs.silverstripe.org for the site. Would it break much to simply rename doc.silverstripe.org to silverstripe-documentation ? Do GitHub repos have some form of redirect we could use ?

**Comment by camfindlay** [comment]
silverstripe-* repo naming convention is for modules, this is a whole project/site hence the current naming. Granted we could change this repo to docs.* however then I'd also have to go change all the deployment settings on the infrastructure too. We can look at that at some point in the future however priorities should be on changes to the project that add value to end users first.

**Comment by SpiritLevel** [comment]
OK :) Just looking for places to polish things up wherever I can.

**Pull request merged**